### PR TITLE
Upgrade ruby and gems

### PR DESCRIPTION
**Pull request by jpawlyn**

### CODE REVIEW

**Review by simonkwesi — APPROVED** [review]
Thanks @jpawlyn

File: Gemfile [hunk 1]
```diff
@@ -7,6 +7,8 @@ gemspec
...
 gem 'rake', '~> 13.0'
 
-gem 'rspec', '~> 3.0'
-
-gem 'rubocop', '~> 0.80'
+group :test do
+  gem 'rspec', '~> 3.0'
+  gem 'vcr'
+  gem 'webmock'
+end
```
Comment (simonkwesi) [inline Gemfile:14]: 🧹

**Comment by jpawlyn** [comment]
> Thanks @jpawlyn

You're very welcome! I can add another one if you like 😃 ! https://github.com/mtomady/icd-api/pull/4

File: spec/icd/api/client_spec.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+RSpec.describe Icd::Api::Client do
+  let(:client) do
+    described_class.new(client_id: '',
+                        client_secret: '')
+  end
+
+  context 'Fetch by Code -- Good Value' do
```
Comment (simonkwesi) [inline spec/icd/api/client_spec.rb:8]: What's a good value? From how I understand it, the value after the dot is used to further specify the disease, it is not required for a code to be valid, though?

```suggestion
  context 'Fetch by Code -- Good Value' do
```

Comment (jpawlyn) [reply spec/icd/api/client_spec.rb:8]: Good question. Hey @KiotoJ 👋🏼 , can you remember? 🙇🏼

Comment (simonkwesi) [reply spec/icd/api/client_spec.rb:8]: Will address this in a follow up PR

**Pull request merged**